csm/core/blogic/const.py

```diff
@@ -952,3 +952,13 @@
 REQUEST_CANCELLED       = 'RequestCancelled'
 SETUP_ERROR             = 'SetupError'
 HTTP_ERROR              = 'HttpError'
+
+# Session Backend factory
+DB                  = 'database'
+IN_MEMORY           = 'in_memory'
+SESSION_STORAGE_KEY = 'CSM>SESSION>storage'
+SESSION_BACKEND_KEY = 'CSM>SESSION>backend'
+LOCAL               = 'local'
+PERSISTENT          = 'persistent'
+IN_MEMORY_DICT      = 'in_memory_dict'
```

Comment:
Why this sort of duplicate variables you have taken ? IN_MEMORY_DICT and PERSISTENT_DB

Reply:
removed